libsrc/rp6502/_localtime.c

```diff
@@ -0,0 +1,31 @@
+#include <rp6502.h>
+#include <time.h>
+#include <stdbool.h>
+
+static bool tzset_set = false;
+
+struct tm* __fastcall__ _time_t_to_tm (const time_t t);
+
+struct tm* __fastcall__ _localtime (const time_t* timep)
+{
+    time_t time = *timep;
+    struct tm* tm;
+    ria_set_axsreg (*timep);
```

Comment:
The following change saves about 10 bytes:
```suggestion
    struct tm* tm;
    time_t time = *timep;
    ria_set_axsreg (time);
```